src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertsArtworks.tsx

```diff
@@ -288,22 +284,25 @@ export const SavedSearchAlertsArtworksQueryRenderer: React.FC<SavedSearchAlertsA
 
 const SavedSearchAlertsArtworksPlaseholderContext: React.FC = () => {
   return (
-    <Join separator={<Spacer y={2} />}>
-      <CriteriaPillsPlaceholder />
-      <SkeletonBox display={["none", "block"]}>
-        <Separator />
-      </SkeletonBox>
-      <Flex flexDirection="column">
-        <SkeletonText variant="sm-display">
-          20 works currently on Artsy match your criteria.
-        </SkeletonText>
-        <SkeletonText variant="sm-display">
-          See our top picks for you:
-        </SkeletonText>
-        <Spacer y={[4, 2]} />
-      </Flex>
-      <ArtworkGridPlaceholder columnCount={2} />
-    </Join>
+    // Setting a max height to force scrolling to top when the content changes.
+    <>
```

Comment:
Not necessary or blocking (maybe quiet and no animation is better) - but if you wrap this in the `<Skeleton>` it will [pulse as a loading](https://github.com/artsy/palette/blob/main/packages/palette/src/elements/Skeleton/Skeleton.tsx#L66-L77) state

Reply:
Thanks 🙏  This component is used and wrapped with `<Skeleton>` [here](https://github.com/artsy/force/pull/13565/files#diff-9acbdd2cc4ae3f70215edaa54eb40dbff5ba88c57080b2cd6e37fda3b21e1666R314). Looks like `SavedSearchAlertsArtworksPlaseholderContext` already 👍